src/ShallowWrapper.js

```diff
@@ -3,6 +3,7 @@ import flatten from 'lodash/flatten';
 import unique from 'lodash/uniq';
 import compact from 'lodash/compact';
 import cheerio from 'cheerio';
+import merge from 'lodash/merge';
```

Comment:
we have `object.assign` available, please use that instead of adding a new dependency.